crates/viewer/re_renderer/src/importer/stl.rs

```diff
@@ -43,14 +42,24 @@ pub fn load_stl_from_buffer(
             .tuples::<(_, _, _)>()
             .map(glam::UVec3::from)
             .collect::<Vec<_>>(),
-        vertex_positions: bytemuck::cast_slice(&triangles).to_vec(),
+
+        vertex_positions: faces
+            .iter()
+            .flat_map(|f| {
```

Comment:
bytemucking this no longer works here? :(
Why? Looks bytemuckable

Reply:
Sadly not, because we need to grab the vertex positions from `vertices` :(

But I am not very familiar with bytemuck, so maybe there's some other secret technique.

Reply:
ohhh wait a sec, I wasn't reading any of this properly  before. This thing has indices?! That's.. kind of bs on the surface since STL doesn't even support vertex indices (it's like the least complex format you can imagine). So I got curious and checked the loader: it _creates_ indices whenever it sees two vertices that are exactly the same.. which happens very very often in STL ofc 🤦 . It's not pretty how it's done imho, but works
https://docs.rs/stl_io/latest/src/stl_io/lib.rs.html#69

.. so what you should be really doing here is to bytemuck the face array into `triangle_indices` and then bytemuck the vertices into `vertex_positions`. And hope that this still works 😄